solidity/scripts/deploy_cloneable.ts

```diff
@@ -8,14 +8,8 @@ export async function deployFungible(tokenName: string) {
   const { deployer, args, libraries } = await verifiersDeployer.deployDependencies();
 
   let zetoFactory;
-  const opts = {
-    kind: 'uups',
-    initializer: 'initialize',
-    unsafeAllow: ['delegatecall']
-  };
```

Comment:
I thought we needed this for upgradable contracts?

Reply:
That's accurate, this options object is needed for upgradeable contracts, but here we are deploying the implementation contract that can then be cloned, so no need for these parameters.

What we could do is, demonstrate how the cloned instance can then be turned into an upgradeable contract. But that's making the sample too complex (first deploy the implementation, then deploy the clone factory, then create the clone, and then make it into an upgradeable).